api/v1/models/index.js

```diff
@@ -8,4 +9,10 @@ module.exports = {
 	Volunteer: require('./Volunteer'),
 	Team: require('./Team'),
 	TeamsUser: require('./TeamsUser')
+=======
```

Comment:
It's just a conflict tag. I don't know how that is still in there (pretty sure I removed it actually) but it's really not that hard to fix...